tree/treeplayer/inc/ROOT/TTreeProcessorMT.hxx

```diff
@@ -46,6 +48,13 @@ the threaded object.
 */
 
 namespace ROOT {
+
+/// A cluster of entries
+struct EntryRange {
```

Comment:
IIUC this introduces a new entity `ROOT::EntryRange` -- as discussed we shouldn't touch the public `ROOT` namespace unless we have a good reason. Can this live somewhere else?

Reply:
Correct me if I am wrong, but it used to reside in the same namespace before. But under the name `EntryCluster`.

I think it should be okay to move it to `ROOT::Internal`.

The only place different that TTreeProcessMT.[h/c]xx where this struct is (now) used is  in `RLoopManager --> RunTreeProcessorMT()`.

Reply:
if I'm not mistaken it used to be in an anonymous namespace in the .cxx file, so it wasn't even visible outside of the source file before

Reply:
> This struct can go inside of the TTreeProcessorMT class (I think we never mentioned that before)

This is the solution I was advocating for the other day, before finding out that it breaks Python compatibility if we put it in the private section of the class. So for that struct to exist at all inside the class, it must be public. Otherwise just put it in Internal

Reply:
I recall now. Okay, so because of python compatibility, we rule out this option now.

To avoid polluting the ROOT or ROOT::Internal namespace with a structure that almost fully duplicates REntryRange, I am considering to make this `EntryRange` a pair.

Reply:
Finally, I sticked to a pair!